fission/package.json

```
@@ -1,25 +1,32 @@
 {
   "name": "synthesis-fission",
   "private": false,
-  "version": "0.0.1",
+  "version": "7.1.0",
   "type": "module",
+  "main": ".vite/build/main.cjs",
+  "author": "Autodesk",
+  "description": "Synthesis",
   "scripts": {
     "init": "(bun run assetpack && bun run playwright:install) || (npm run assetpack && npm run playwright:install)",
     "host": "vite --open --host",
     "dev": "vite --open",
+    "test": "vitest",
     "build": "tsc && vite build",
     "build:prod": "tsc && vite build --base=/fission/ --outDir dist/prod",
     "build:dev": "tsc && vite build --base=/fission-closed/ --outDir dist/dev",
     "preview": "vite preview --base=/fission/",
-    "test": "vitest",
     "lint": "bunx --bun biome lint",
     "lint:fix": "bunx --bun biome lint --fix",
     "fmt": "bunx --bun biome format src",
     "fmt:fix": "bunx --bun biome format src --write",
     "style": "bun run fmt && bun run lint",
     "style:fix": "bun run fmt:fix && bun run lint:fix",
     "assetpack": "git lfs pull && tar -xf public/assetpack.zip -C public/",
-    "playwright:install": "bun x playwright install"
+    "playwright:install": "bun x playwright install",
+    "start": "electron-forge start",
+    "package": "electron-forge package",
+    "make": "electron-forge make",
+    "publish": "electron-forge publish"
```

```suggestion
    "electron:start": "electron-forge start",
    "electron:package": "electron-forge package",
    "electron:make": "electron-forge make",
    "electron:publish": "electron-forge publish"
```

I think it makes sense to prefix these all with electron just for clarity